loopy/codegen/bounds.py

```diff
@@ -55,57 +55,57 @@ def get_approximate_convex_bounds_checks(domain, check_inames, implemented_domai
 
 # {{{ on which inames may a conditional depend?
 
-def get_usable_inames_for_conditional(kernel, sched_index):
+def get_usable_inames_for_conditional(kernel, sched_indices):
```

Comment:
Add documentation on what the multiple inputs mean and what's supposed to be returned.